### ISSUE TITLE
Grafana 8.4.3 + grafana-wtf 0.13.1 -> "TypeError: '<' not supported between instances of 'dict' and 'dict'"

### ISSUE THREAD

**Issue by IgorOhrimenko** [panodata/grafana-wtf#32]
grafana-wtf 0.13.1
Grafana 8.4.3

grafana-wtf explore datasources --format=yaml
```
2022-03-06 09:33:33,459` [grafana_wtf.core      ] INFO   : Setting up response cache to expire after 300 seconds
2022-03-06 09:33:33,462 [grafana_wtf.core      ] INFO   : Scanning dashboards
2022-03-06 09:33:33,469 [grafana_wtf.core      ] INFO   : Found 133 dashboards
  0%|                                                                                                                                                                                     | 0/133 [00:00<?, ?it/s]2022-03-06 09:33:33,475 [grafana_wtf.core      ] INFO   : Fetching dashboards in parallel with 5 concurrent requests
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 133/133 [00:00<00:00, 453.97it/s]2022-03-06 09:33:34,067 [grafana_wtf.core      ] INFO   : Scanning datasources
2022-03-06 09:33:34,069 [grafana_wtf.core      ] INFO   : Found 21 data sources
Traceback (most recent call last):
  File "/usr/local/bin/grafana-wtf", line 10, in <module>
    sys.exit(run())
  File "/usr/local/lib/python3.7/dist-packages/grafana_wtf/commands.py", line 234, in run
    results = engine.explore_datasources()
  File "/usr/local/lib/python3.7/dist-packages/grafana_wtf/core.py", line 390, in explore_datasources
    ix = Indexer(engine=self)
  File "/usr/local/lib/python3.7/dist-packages/grafana_wtf/core.py", line 478, in __init__
    self.index()
  File "/usr/local/lib/python3.7/dist-packages/grafana_wtf/core.py", line 481, in index
    self.index_dashboards()
  File "/usr/local/lib/python3.7/dist-packages/grafana_wtf/core.py", line 513, in index_dashboards
    ds_panels = self.collect_datasource_items(dbdetails.panels)
  File "/usr/local/lib/python3.7/dist-packages/grafana_wtf/core.py", line 494, in collect_datasource_items
    return sorted(items)
TypeError: '<' not supported between instances of 'dict' and 'dict'
```

**Comment by carpenterbees** [comment]
Same error for me. How can it be fixed?

**Comment by amotl** [comment]
Dear Igor and Oleksandr,

thank you for the reports. I will try to look into the issue. Do you have any clue what might have changed with Grafana 8.4.3?

With kind regards,
Andreas.

**Comment by amotl** [comment]
Dear Igor and Oleksandr,

I have been able to fix the flaw you reported and just released version 0.13.2 including this improvement.

Thank you again and with kind regards,
Andreas.
